ui/test-file/test-indicator.tsx

```diff
@@ -21,6 +21,8 @@ export default function TestIndicator({ status, describe }: Props) {
   if (!describe) {
     if (status === "passed") {
       Icon = CheckCircle;
+    } else if (status === "todo") {
```

Comment:
I thought the `status` would be passed or failed so little confused how it could be "todo".

Reply:
Well I wanted to set todo's status as todo's so I could parse them on the sidebar.

A test todo is not either passed or failed, it's skipped. But change as you wish!

Reply:
I'm happy to leave it as is if it works. I'll get this merged today. Thank you for taking the time to implement this. Appreciate it 💯

Reply:
Hey actually you made me realize when cleaning up i messed up and didnt commit necessary files to show up the todo's, all fixed and working on new commit, I could try and add some tests later tomorrw

Reply:
No worries 👍